### ISSUE TITLE
[Suggestion] Add rbatis tag on stackoverflow

### ISSUE THREAD

**Issue by TDiblik** [rbatis/rbatis#171]
Hi, I was going through some issues in this repo, and I've seen some issues taged with the `question` tag. I believe, that SEO wise and managment wise, it would be beneficial to start a new stackoverflow tag.

Why?
- More issues focused on development of this project and less issues as questions
- When searching for answear, stackoverflow is usually the first suggestion
- Issues would be in history in one place, ordered

Why not?
- There are not a lot of questions on github
- Two different platforms for one project

I have not contributed to this project in any way, shape or form, however, I am learning it and using it in my personal projects. I think, from outsider perspective, this would be a nice improvement.

**Comment by zhuxiujia** [comment]
> 

Thanks for the advice and if you have enough time you can copy the problem to StackOverflow.

**Comment by zhuxiujia** [comment]
> Hi, I was going through some issues in this repo, and I've seen some issues taged with the `question` tag. I believe, that SEO wise and managment wise, it would be beneficial to start a new stackoverflow tag.
> 
> Why?
> 
> * More issues focused on development of this project and less issues as questions
> * When searching for answear, stackoverflow is usually the first suggestion
> * Issues would be in history in one place, ordered
> 
> Why not?
> 
> * There are not a lot of questions on github
> * Two different platforms for one project
> 
> I have not contributed to this project in any way, shape or form, however, I am learning it and using it in my personal projects. I think, from outsider perspective, this would be a nice improvement.

Thanks for the advice and if you have enough time you can copy the problem to StackOverflow.

**Comment by TDiblik** [comment]
Yeah, why not :D, however I currentlly do not have enought points to create tags... Once somebody creates the tag, I'll be more than happy to copy them (questions) to StackOverflow.
